packages/now-cli/test/integration.js

```diff
@@ -539,7 +539,7 @@ test('Deploy `api-env` fixture and test `vercel env` command', async t => {
   async function nowEnvAddSystemEnv() {
     const now = execa(
       binaryPath,
-      ['env', 'add', 'system', 'VERCEL_URL', ...defaultArgs],
+      ['env', 'add', 'system', 'NEXT_PUBLIC_VERCEL_URL', ...defaultArgs],
```

Comment:
Updated this test to properly test that references to system env variables are correctly functioning (with `vc dev`, `vc env pull`, etc...).